Index: mainV2.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- mainV2.py	(date 1487193038000)
+++ mainV2.py	(revision )
@@ -33,6 +33,7 @@
 
 
 '''
+
 plt.figure()
 plt.plot(patient1.static["MVC2"][:, 0])
 plt.plot(np.linspace(0, len(patient1.static["MVC2"][:, 0]), len(patient1.staticRMS["MVC2"][:, 0])), patient1.staticRMS["MVC2"][:, 0])
